bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -2381,6 +2381,17 @@ void ReplicaImp::onMessage<CheckpointMsg>(CheckpointMsg *msg) {
         if (numRelevantAboveWindow >= config_.getfVal() + 1) {
           LOG_INFO(GL, "Number of stable checkpoints above window: " << numRelevantAboveWindow);
           askForStateTransfer = true;
+
+          // For debug
```

Comment:
Are we planning to keep this?

Reply:
I think it is very useful. It is rare so it's not a problem to print it, it is called once only before entering ST, and it gives lots of information why we are entering ST.
Without this, it is much harder to understand why we enter ST at the 1st place. This one coveres only one of the cases and gives you full view on the reason.